Thanks to queueRAM for finding the solution to the linking issue!
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@
 CFLAGS = -O2
 CXXFLAGS = -O2
 WXCXXFLAGS= `$(WXCONFIG) --cxxflags` -Iudis86 -Imhash/include -MMD -O2 -c ${OPTFLAGS}
-WXLDFLAGS = `$(WXCONFIG) --libs`
+WXLDFLAGS = `$(WXCONFIG) --libs std,aui`
 RC = `$(WXCONFIG) --rescomp`
 #RC = x86_64-w64-mingw32-windres --define WX_CPU_AMD64
 RCFLAGS = `$(WXCONFIG) --cxxflags | sed s/' '-m.*//g;`
